enterprise/frontend/src/metabase-enterprise/auth/index.js

```diff
@@ -43,22 +43,23 @@ PLUGIN_ADMIN_SETTINGS_UPDATES.push(sections =>
       display_name: t`Enable Password Authentication`,
       description: t`When enabled, users can additionally log in with email and password.`,
       type: "boolean",
-      getHidden: settings =>
-        !settings["google-auth-enabled"] &&
-        !settings["ldap-enabled"] &&
-        !settings["saml-enabled"] &&
-        !settings["jwt-enabled"],
+      getHidden: (_, derivedSettings) =>
```

Comment:
`settings` contains values only explicitly set by the user in the UI that are no defaults. In the following cases the value for setting would be `null`:
- default value
- set by an environment variable

`derivedSettings` contains computed values, e.g. it will have the actual value if it's the default one etc.

Reply:
Fixed

Reply:
we don't use lodash btw :)